apps/bfd-pipeline/bfd-pipeline-shared-test-utils/src/main/java/gov/cms/bfd/pipeline/PipelineTestUtils.java

```diff
@@ -224,26 +223,18 @@ public void truncateTablesInDataSource() {
   }
 
   /**
-   * For compatibility with HSQLDB and Postgresql, all schema names must have case preserved but any
-   * quotes in the name must be removed.
+   * Normalize the schema names by removing any quotes.
    *
-   * @param connection the connection
    * @param schemaNameSpecifier name of a schema from a hibernate annotation
    * @return value compatible with call to {@link Connection#setSchema(String)}
    * @throws SQLException the sql exception
    */
-  private String normalizeSchemaName(Connection connection, String schemaNameSpecifier)
-      throws SQLException {
-    String sql = schemaNameSpecifier.replaceAll("`", "");
-    if (isSchemaNameUppercaseRequired(connection)) {
-      sql = sql.toUpperCase();
-    }
-    return sql;
+  private String normalizeSchemaName(String schemaNameSpecifier) throws SQLException {
```

Comment:
we can probably think about removing the necessity to check for quotes...probably in another ticket tho'.